lib/ex_prosemirror/block/hard_break.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ExProsemirror.Block.HardBreak do
+  @moduledoc ~S"""
+  Hard Break tag
+  """
+
+  use ExProsemirror.Schema
+  use ExProsemirror.Encoder.Json, type: :hard_break
+
+  import Ecto.Changeset
+
+  @doc false
+  embedded_schema do
+  end
+
+  @doc false
+  def changeset(struct_or_changeset, attrs \\ %{}) do
+    struct_or_changeset
+    |> cast(attrs, [])
+  end
+
+  defimpl ExProsemirror.Encoder.HTML do
+    import Phoenix.HTML.Tag, only: [tag: 1]
+
+    def encode(%{attrs: _attrs}, _opts) do
+      tag(:br)
+    end
+  end
+
+  def extract_simple_text(_struct), do: ""
```

Comment:
Maybe for simple_text we should return \n ?
```suggestion
  def extract_simple_text(_struct), do: "\n"
```